polybar-scripts/dunst-controller/dunstctl.sh

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env sh
+
+unmute() {
+    dunstctl set-paused false && \
+        dunstify 'dunst' 'Unmuted notifications.' -i polari -u low &
+}
+
+mute() {
+    dunstify 'dunst' 'Muting notifications...' -i polari -u low &
+    # We're about to mute notifications, so the above notification will
+    # effectively not show, since set-paused hides notifications. So, sleep
+    # for 2 seconds.
+    sleep 2
+    # Close all to prevent notification from showing up once we unmute.
+    dunstctl close-all
+    dunstctl set-paused true
+}
+
+pop_all() {
+    for id in $(dunstctl history | while jq -r '.data[][].id.data'); do
+        dunstctl history-pop $id
+    done
```

Comment:
Are you sure this works? Did you forget to commit a change? This bit of code is not valid.

Here is how it should be

```suggestion
    dunstctl history | jq '.data[][].id.data' | while IFS= read -r id; do
        dunstctl history-pop "$id"
    done
```

Reply:
Right, my mistake, no, hadn't advertised it in the README either, but I did leave the double-click in `dunstctl.ini`. I use the `fish` scripts personally. Thanks for the fix, will include it in the README as well.